src/psyclone/tests/psyad/main_test.py

```diff
@@ -43,6 +43,53 @@
 import pytest
 from psyclone.psyad import main
 
+
+TEST_PROG = (
+    "program test\n"
+    "integer :: a\n"
+    "a = 0\n"
+    "end program test\n")
+
+# TODO we currently have no way of automatically capturing which variables
```

Comment:
Should reference an issue

Reply:
Actually, it was outdated as the harness generation no longer worries about this - it simply does the inner product for every kernel argument which I think will work (since those variables that aren't active won't be modified anyway).